web-server/v0.4/README.md

```diff
@@ -76,6 +76,7 @@ export default {
   '/dev/datastoreConfig': {
       "elasticsearch": "http://elasticsearch.example.com",
       "results": "http://results.example.com",
+      "server": "http://server.example.com",
```

Comment:
Did you consider naming this `graphql`, or something like that, instead of using the generic name of `server`?

Reply:
Yes, `server` is too generic here - I've replaced this with `graphql`.